.changelog/9589.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+server: Fixes a panic introduced in 1.9.0 where servers in a specific state may crash during node de-registration.
```

Comment:
Could we be more specific -> something like "fixes a panic in 1.9.0 in clusters that have consul connect enabled with <details about specific state> during re-registration of nodes."  ?

Reply:
I've made it more specific. Open to further suggestions?